os/os.h

```diff
@@ -24,7 +24,7 @@ enum {
 	os_windows,
 	os_android,
 	os_dragonfly,
-
```

Comment:
Please leave the empty space, it's to denote the end marker rather than a real entry.